<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2018-12-01T23:19:12" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="Checkout cypress" timestamp="2018-12-01T23:19:12" tests="2" failures="2" time="0">
    <testcase name="Checkout cypress tests coolboard" time="0" classname="tests coolboard">
      <failure><![CDATA[CypressError: cy.visit() failed trying to load:

https://improve-ui-for-e2e-testing.coolboard.fun/boards

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: getaddrinfo ENOTFOUND improve-ui-for-e2e-testing.coolboard.fun improve-ui-for-e2e-testing.coolboard.fun:443

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

The stack trace for this error is:

Error: getaddrinfo ENOTFOUND improve-ui-for-e2e-testing.coolboard.fun improve-ui-for-e2e-testing.coolboard.fun:443
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)

    at Object.cypressErr (http://localhost:64708/__cypress/runner/cypress_runner.js:63008:11)
    at Object.throwErr (http://localhost:64708/__cypress/runner/cypress_runner.js:62973:18)
    at Object.throwErrByPath (http://localhost:64708/__cypress/runner/cypress_runner.js:63000:17)
    at http://localhost:64708/__cypress/runner/cypress_runner.js:53634:31
    at visitFailedByErr (http://localhost:64708/__cypress/runner/cypress_runner.js:53245:12)
    at http://localhost:64708/__cypress/runner/cypress_runner.js:53633:22
    at tryCatcher (http://localhost:64708/__cypress/runner/cypress_runner.js:124249:23)
    at Promise._settlePromiseFromHandler (http://localhost:64708/__cypress/runner/cypress_runner.js:122267:31)
    at Promise._settlePromise (http://localhost:64708/__cypress/runner/cypress_runner.js:122324:18)
    at Promise._settlePromise0 (http://localhost:64708/__cypress/runner/cypress_runner.js:122369:10)
    at Promise._settlePromises (http://localhost:64708/__cypress/runner/cypress_runner.js:122444:18)
    at Async._drainQueue (http://localhost:64708/__cypress/runner/cypress_runner.js:119173:16)
    at Async._drainQueues (http://localhost:64708/__cypress/runner/cypress_runner.js:119183:10)
    at Async.drainQueues (http://localhost:64708/__cypress/runner/cypress_runner.js:119057:14)
    at <anonymous>]]></failure>
    </testcase>
    <testcase name="Checkout cypress logged-in user can add list and card" time="0" classname="logged-in user can add list and card">
      <failure><![CDATA[CypressError: cy.visit() failed trying to load:

https://improve-ui-for-e2e-testing.coolboard.fun/boards

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: getaddrinfo ENOTFOUND improve-ui-for-e2e-testing.coolboard.fun improve-ui-for-e2e-testing.coolboard.fun:443

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

The stack trace for this error is:

Error: getaddrinfo ENOTFOUND improve-ui-for-e2e-testing.coolboard.fun improve-ui-for-e2e-testing.coolboard.fun:443
    at errnoException (dns.js:50:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:92:26)

    at Object.cypressErr (http://localhost:64708/__cypress/runner/cypress_runner.js:63008:11)
    at Object.throwErr (http://localhost:64708/__cypress/runner/cypress_runner.js:62973:18)
    at Object.throwErrByPath (http://localhost:64708/__cypress/runner/cypress_runner.js:63000:17)
    at http://localhost:64708/__cypress/runner/cypress_runner.js:53634:31
    at visitFailedByErr (http://localhost:64708/__cypress/runner/cypress_runner.js:53245:12)
    at http://localhost:64708/__cypress/runner/cypress_runner.js:53633:22
    at tryCatcher (http://localhost:64708/__cypress/runner/cypress_runner.js:124249:23)
    at Promise._settlePromiseFromHandler (http://localhost:64708/__cypress/runner/cypress_runner.js:122267:31)
    at Promise._settlePromise (http://localhost:64708/__cypress/runner/cypress_runner.js:122324:18)
    at Promise._settlePromise0 (http://localhost:64708/__cypress/runner/cypress_runner.js:122369:10)
    at Promise._settlePromises (http://localhost:64708/__cypress/runner/cypress_runner.js:122444:18)
    at Async._drainQueue (http://localhost:64708/__cypress/runner/cypress_runner.js:119173:16)
    at Async._drainQueues (http://localhost:64708/__cypress/runner/cypress_runner.js:119183:10)
    at Async.drainQueues (http://localhost:64708/__cypress/runner/cypress_runner.js:119057:14)
    at <anonymous>]]></failure>
    </testcase>
  </testsuite>
</testsuites>